### PR TITLE
Improve cron jobs

### PR DESCRIPTION
**Pull request by thibaultleouay**

### CODE REVIEW

File: apps/web/src/app/api/checker/cron/10m/route.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { cron } from "../_cron";
+
+export const runtime = "edge";
+export const preferredRegion = ["auto"];
```
Comment (mxkaske) [inline apps/web/src/app/api/checker/cron/10m/route.ts:4]: Nice to have it here, even if default. Reminds us!

File: apps/web/src/app/api/checker/regions/_checker.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import { Receiver } from "@upstash/qstash/cloudflare";
```
Comment (mxkaske) [inline apps/web/src/app/api/checker/regions/_checker.ts:1]: What's the benefit over using `@upstash/qstash/nextjs`?

File: apps/web/src/app/api/checker/regions/arn1/route.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { checker } from "../_checker";
+
+export const runtime = "edge";
+export const preferredRegion = ["arn1"];
+export const dynamic = "force-dynamic";
+
+export async function POST(request: Request) {
```
Comment (mxkaske) [inline apps/web/src/app/api/checker/regions/arn1/route.ts:7]: We could move it to a `GET` method and allow the `url` to be a search param.

File: packages/tinybird/src/client.ts [hunk 4]
```diff
@@ -4,18 +4,24 @@ import * as z from "zod";
...
+  workspaceId: z.string(),
+  pageId: z.string(),
+  monitorId: z.string(),
+  timestamp: z.number().int(),
+  statusCode: z.number().int(),
+  latency: z.number().int(), // in ms
+  url: z.string(),
+  metadata: z.string().optional().default(""),
+  region: z.string().min(4).max(4), // TODO: object + stringify json
+});
```
Comment (mxkaske) [inline packages/tinybird/src/client.ts:18]: We will probably have two different schemas for the `publishPingResponse` (datasource) and `getResponseList` (pipe). 

We don't have to return all the values, could group the entries,.. or transform the values e.g.

```
metadata: z.string().transform((m) => JSON.parse(m))
```
while saving it as string.

**Review by mxkaske — COMMENTED** [review]
Good stuff, man!

File: apps/web/src/app/api/checker/regions/arn1/route.ts (hunk 3 above)
Comment (mxkaske) [reply apps/web/src/app/api/checker/regions/arn1/route.ts:7]: Somehow, I think `POST` is cleaner. Having the props inside the `request.body` is cleaner.

File: apps/web/src/app/api/checker/regions/_checker.ts (hunk 2 above)
Comment (thibaultleouay) [reply apps/web/src/app/api/checker/regions/_checker.ts:1]: We are in the edge environment which is the same as cloudfare worker 

We cant use all the node api

File: apps/web/src/app/api/checker/regions/arn1/route.ts (hunk 3 above)
Comment (thibaultleouay) [reply apps/web/src/app/api/checker/regions/arn1/route.ts:7]: Qstash post a Json to an endpoint :)

File: packages/tinybird/src/client.ts (hunk 4 above)
Comment (thibaultleouay) [reply packages/tinybird/src/client.ts:18]: Sure just extract it when I did some small update

**Review by mxkaske — APPROVED** [review]

**Pull request merged**